node/peer_states.js

```diff
@@ -90,7 +90,7 @@ TChannelPeerHealthyState.prototype.type = 'tchannel.healthy';
 
 TChannelPeerHealthyState.prototype.toString = function healthyToString() {
     var self = this;
-    return '[HealthyPeer ' + self.okCount + 'ok ' + self.notOkCount + 'err]';
+    return '[HealthyPeer ' + self.okCount + ' ok ' + self.notOkCount + ' err]';
```

Comment:
Mistakes like this are easy to make when you don't use something like util.format...